compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -6407,7 +6407,6 @@ private BType checkInvocationArgs(BLangInvocation iExpr, List<BType> paramTypes,
 
         BType retType = typeParamAnalyzer.getReturnTypeParams(env, bInvokableType.getReturnType());
         if (restType != symTable.semanticError &&
```

Comment:
Not related to this PR, but this should probably be retType instead of restType, right?

Reply:
Yes